guide/build-config.md

```diff
@@ -27,6 +27,7 @@ version: 1.0.0
 milestone: DR1
 brewTagPrefix: fb-1.0-pnc # Optional
 group: Test Product 1.0-all
+releaseStorageUrl: http://testurl.com/releaseStorage
```

Comment:
`brewTagPrefix` above is marked as optional. Is this optional? What about any others?

Reply:
you don't ever need to set up brewTagPrefix that's why I think it is highlighted as optional, others are not optional in this sense and their description when they are used and user needs to set them is below.

Reply:
So are you saying its now mandatory to provide `releaseStorageUrl` in the yaml ?

Reply:
Yes, it's needed if you want to run `pig release` or `pig configure/run` when using automatic milestone increment meaning you use `*` in milestone